### ISSUE TITLE
Research Item Hold Request, insufficient color contrast

### ISSUE THREAD

**Issue by wlla** [NYPL/design-toolkit#180]
Link to the item record doesn't have enough contrast, foreground #176e91; background #d7d4d0

Research Item Hold Request page: http://dev-discovery.nypl.org/hold/request/b11138602-i8640935

**Comment by ricardoom** [comment]
making the bread crumb links & static text `$nypl-dark-gray`to get around this issue. also adding a hover state for extra snazz

**Comment by EdwinGuzman** [comment]
Updated link: http://dev-discovery.nypl.org/research/collections/shared-collection-catalog/hold/request/b11138602-i8640935
